tests/test_spatial_resample.py

```diff
@@ -142,7 +142,7 @@ def test_flips(self, img, device, data_param, expected_output):
                 img = img.to(device)
             out = SpatialResample()(img=img, **data_param)
             assert_allclose(out, expected_output, rtol=1e-2, atol=1e-2)
-            assert_allclose(out.affine, data_param["dst_affine"])
```

Comment:
May I know why remove the verification for predefined `data_param["dst_affine"]`?

Thanks.

Reply:
Hi @Nic-Ma, we don't remove this verification. We added `to_affine_nd` to make the dimension right. If the data is not metatensor, we will add a fake affine.